src/Twilio/IncomingPhoneNumbers.hs

```diff
@@ -8,26 +8,32 @@ module Twilio.IncomingPhoneNumbers
   , PostIncomingPhoneNumber(..)
   , Twilio.IncomingPhoneNumbers.get
   , Twilio.IncomingPhoneNumbers.post
+  , VoiceMethod(..)
   ) where
 
 import Control.Applicative
 import Control.Monad.Catch
 import Data.Aeson
-import Data.Text (Text)
+import Data.Text (Text, pack)
 import Data.Text.Encoding
+import Data.Maybe (catMaybes)
 
 import Control.Monad.Twilio
 import Twilio.IncomingPhoneNumber
 import Twilio.Internal.Request
 import Twilio.Internal.Resource as Resource
 import Twilio.Types
 
+data VoiceMethod = Get | Post deriving Show
+
 {- Resource -}
 
 data PostIncomingPhoneNumber = PostIncomingPhoneNumber {
     pipnPhoneNumber :: Text,
     pipnFriendlyName :: Maybe Text,
-    pipnSmsApplicationSID :: ApplicationSID
+    pipnSmsApplicationSID :: ApplicationSID,
+    pipnVoiceMethod :: VoiceMethod,
+    pipnVoiceUrl :: Maybe Text
```

Comment:
I had written this same code, except that i put the voice method inside the maybe as well. It doesn't rwally make sense if there is no voice url. But, its not that big of a deal.